config/locales/en-GB.yml

```diff
@@ -238,10 +238,20 @@ en-GB:
       require_customer_login: "This shop is for customers only."
       require_login_html: "Please %{login} if you have an account already. Otherwise, %{register} to become a customer."
       require_customer_html: "Please %{contact} %{enterprise} to become a customer."
-  invoice_column_item: "Item"
-  invoice_column_qty: "Qty"
   invoice_column_tax: "VAT"
   invoice_column_price: "Price"
+  invoice_column_item: "Item"
+  invoice_column_qty: "Qty"
+  invoice_column_unit_price_with_taxes: "Unit price (Incl. tax)"
+  invoice_column_unit_price_without_taxes: "Unit price (Excl. tax)"
+  invoice_column_price_with_taxes: "Total price (Incl. tax)"
+  invoice_column_price_without_taxes: "Total price (Excl. tax)"
+  invoice_column_tax_rate: "Tax rate"
+  tax_invoice: "INVOICE"
+  tax_total: "Total tax (%{rate}):"
+  total_excl_tax: "Total (Excl. tax):"
+  total_incl_tax: "Total (Incl. tax):"
+  invoice_tax_total: "VAT Total:"
```

Comment:
are these keys in `en.yml` too? AFAIK we should only touch `en.yml` to not break Transifex process.

cc/ @mkllnk